### PR TITLE
CHANGELOG: update changelog for 2.0.3

### PR DESCRIPTION
**Pull request by nolouch**
@lilin90 @disksing  PTAL

### CODE REVIEW

File: server/join_test.go [hunk 1]
```diff
@@ -283,6 +283,7 @@ func (s *testJoinServerSuite) TestDeletedPDJoinsPreviousCluster(c *C) {
 
 	cfgs[target].InitialCluster = ""
 	_, err := startPdWith(cfgs[target])
+	fmt.Println(":Debug", err)
```
Comment (disksing) [inline server/join_test.go]: Accident update?

Comment (nolouch) [reply server/join_test.go]: done.

**Comment by siddontang** [comment]
LGTM

**Comment by disksing** [comment]
@CaitinChen PTAL.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+* Fix the issue that adds operator on the stale region
+* Fix the issue that panic when collecting the hot-cache metrics under some condition
+## v2.0.2
+### Improvements
+* Make the balance leader scheduler filter the disconnected nodes
+* Make the tick interval of patrol Regions configurable
+* Modify the timeout of the transfer leader operator to 10s
+### Bug Fixes
+* Fix the issue that the label scheduler does not schedule when the cluster Regions are in an unhealthy state
+* Fix the improper scheduling issue of evict leader scheduler
```
Comment (CaitinChen) [inline CHANGELOG.md]: Fix the improper scheduling issue of `evict leader scheduler`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # PD Change Log
 
+## v2.0.3
+### Bug Fixes
+* Fix the issue that adds operator on the stale region
```
Comment (CaitinChen) [inline CHANGELOG.md]: If this sentence means "修复对旧的 Region 产生调度的问题", change it to "Fix the issue about scheduling of the obsolete Regions"

**Pull request merged**